### ISSUE TITLE
Too long extrusion prevented, when printing more then 1 model

### ISSUE THREAD

**Issue by Gwebber-Net** [bigtreetech/BIQU-BX#18]
When i am printing more then 1 model, always after a couple of layers, the error message pops up,
"Too long extrusion prevented".

And the most funny thing,
After that error, for some weird reason, my printer starts to print, an entirely different model, ontop of what is already printed.
So i was printing a Steppermount, and a benchy was printing halfway through.

**Comment by radek8** [comment]
try to increase the value
#define EXTRUDE_MAXLENGTH

**Comment by Gwebber-Net** [comment]
Yes, I figured that would help,. But I wasnt interested inin going into the firmware for a brand new printer.

I try to not tinker with it.



Outlook voor Android<https://aka.ms/AAb9ysg> downloaden
________________________________
